src/backend/joanie/core/models/courses.py

```diff
@@ -164,7 +164,65 @@ class Organization(parler_models.TranslatableModel, BaseModel):
     )
     signature = models.ImageField(_("signature"), blank=True)
     logo = ThumbnailerImageField(_("logo"), blank=True)
-    country = CountryField(_("country"), default=settings.JOANIE_DEFAULT_COUNTRY_CODE)
+    country = CountryField(
+        _("country"),
+        help_text=_(
+            "Country field will be deprecated soon in order to be replaced by address field."
+        ),
+        default=settings.JOANIE_DEFAULT_COUNTRY_CODE,
+    )
+    address = models.OneToOneField(
```

Comment:
Sorry if already discussed, but shouldn't we allow organizations to have multiple addresses ?

Reply:
We should place the relation on the `Address` model after some thoughts. This means that will adopt the same behavior for `Organization` relationship on managing addresses as it is for a `User`.